views/confirm.hbs

```diff
@@ -35,7 +35,8 @@
             </div>
             <p class='text'>Please check your inbox (and spam) and click on the link we sent to verify your account.</p>
             <p class='retry-text'>Think it's taking too long? .edu emails often have some delay to them, but feel free to have us try again.</p>
-            <button>RESEND EMAIL</button>
+            <button class='resend-verification'>RESEND EMAIL</button>
+            <div style='height: 20px;'></div>
```

Comment:
Why use an empty `div` here instead of just adding a top margin to `.response`? CSS would be more consistent and less confusing to people reading the code in the future.